### DISCUSSION TITLE
Is there a place that shows benchmark comparisons between versions?

### DISCUSSION THREAD

**Discussion by tirithen** [lit/lit#4724] [Q&A] [answered]
Hi! I wonder if there is a place where you publish benchmark comparisons between versions?

In a project I work on due to dependency reasons (multiple versions of lit in use) and time constraints I currently use an older version of lit that I wish, and I wonder if there is a place to see hints on possible performance gains after doing the upgrade work.

I could see that tachometer was used for benchmarking so I wondered if you save/publish these reports somewhere?

**Comment by augustjk** [comment, marked as answer]
I don't know of any published benchmarks but it's possible to checkout to a tag and run `npm run benchmarks` to run the tachometer setup for that version.

I don't believe there are any significant changes in performance between v2 and v3 though.

**Reply by tirithen** [reply]
Thanks for the answer!
